File: testprojects/tests/java/org/pantsbuild/testproject/testjvms/TestBase.java
// Copyright 2015 Pants project contributors (see CONTRIBUTORS.md).
// Licensed under the Apache License, Version 2.0 (see LICENSE).

package org.pantsbuild.testproject.testjvms;

import static org.junit.Assert.assertEquals;

/**
 * Base class for testing what version of java is running.
 * */
public class TestBase {
  public void assertJavaVersion(String expected) {
    String version = System.getProperty("java.version");
    version = version.substring(0, version.indexOf('.', version.indexOf('.')+1));
    assertEquals(expected, version);
  }
}

File: tests/python/pants_test/test_base.py
# Copyright 2018 Pants project contributors (see CONTRIBUTORS.md).
# Licensed under the Apache License, Version 2.0 (see LICENSE).

from pants.testutil.test_base import AbstractTestGenerator as AbstractTestGenerator  # noqa
from pants.testutil.test_base import TestBase as TestBase  # noqa
from pants_test.deprecated_testinfra import deprecated_testinfra_module


deprecated_testinfra_module()
